cli/package.json

```diff
@@ -47,6 +47,7 @@
     "cli-table3": "^0.6.4",
     "dotenv": "^16.4.5",
     "nochoices": "^1.1.4",
+    "node-fetch": "^3.3.2",
```

Comment:
Is there a particular reason we have this dependency here? Node v20 should have fetch built-in

Reply:
Not a good reason. Removing.